pkg/iptables/iptables.go

```diff
@@ -99,10 +137,40 @@ func ConfigureFirewall(firewallConfiguration FirewallConfiguration) error {
 	return nil
 }
 
+func (fc FirewallConfiguration) cleanupRules(commands []*exec.Cmd) []*exec.Cmd {
+	// delete ref from prerouting
+	commands = append(
+		commands,
+		fc.makeJumpFromChainToAnotherForAllProtocols(
+			IptablesPreroutingChainName,
+			redirectChainName,
+			"install-proxy-init-prerouting",
+			true))
+
+	// delete ref from output
+	commands = append(
+		commands,
+		fc.makeJumpFromChainToAnotherForAllProtocols(
+			IptablesOutputChainName,
+			outputChainName,
+			"install-proxy-init-output",
+			true))
+
+	// flush chains
+	commands = append(commands, fc.makeFlushChain(outputChainName))
+	commands = append(commands, fc.makeFlushChain(redirectChainName))
+
+	// delete chains
+	commands = append(commands, fc.makeDeleteChain(outputChainName))
+	commands = append(commands, fc.makeDeleteChain(redirectChainName))
+
+	return commands
+}
+
 // formatComment is used to format iptables comments in such way that it is possible to identify when the rules were added.
 // This helps debug when iptables has some stale rules from previous runs, something that can happen frequently on minikube.
 func formatComment(text string) string {
-	return fmt.Sprintf("proxy-init/%s/%s", text, ExecutionTraceID)
+	return fmt.Sprintf("proxy-init/%s", text)
```

Comment:
Now that the timestamp is not included, this function comment should be amended to not make mention of "when the rules were added".